docs/troubleshooting.md

```diff
@@ -386,4 +386,10 @@ You can also remove the `hwe` kernel and use the `generic` one: this way, the pr
 
 :::tip
 Alternatively, in a fresh Ubuntu 18.04 install, you can switch to UEFI and you won't have this issue.
-:::
\ No newline at end of file
+:::
+
+## Extended characters are not supported in SMB paths, user names, and passwords.    
```

Comment:
This title will be displayed in the left menu of the doc, so a shorter title would be better eg "Extended characters support"

And more description after the title. Thanks! :+1:

Reply:
Of course.